spec/index.html

```diff
@@ -2437,7 +2437,27 @@ <h2 id="media-type-registration">Media Type Registration</h2>
             <dt>Required parameters:</dt>
             <dd>N/A</dd>
             <dt>Optional parameters:</dt>
-            <dd>N/A</dd>
+            <dd>
+              <dl>
+                <dt>charset</dt>
+                <dd>
+                  The <code>charset</code> parameter may be provided to definitively
+                  specify the document's character encoding,
+                  overriding any character encoding declarations in the document.
+                  The parameter's value must be one of the labels of the character
+                  encoding used to serialize the file.
+                </dd>
+                <dt>projection</dt>
+                <dd>
+                  The <code>projection</code> parameter may be provided to definitively
+                  specify the document's associated Tiled Coordinate Reference System (TCRS),
+                  overriding any projection declarations in the document.
+                  The value may be one of the defined
+                  <a href="#tiled-coordinate-reference-systems-table">TCRS identifiers</a>
+                  or a custom projection identifier.
+                </dd>
```

Comment:
@prushforth Is this accurately describing the `projection` parameter?

Reply:
I have since become less convinced of the value of media type parameters to convey semantics like projection, and I think it might be worth discussing to remove the parameter altogether.  I try to avoid sending it where I notice I used to do so.

Reply:
Okay I'll remove the `projection` parameter entirely. Filed https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/436 so that others inspecting the code may not wonder why it isn't defined in the spec.